From 082ce10dd191f036f5a6e1fb31134e9736ba3643 Mon Sep 17 00:00:00 2001
From: Antonios Christidis <a-christidis@ti.com>
Date: Fri, 17 Jan 2025 15:05:39 -0600
Subject: [PATCH] Ignore Compiler Warnings


Within the cmake build system, the compiler -Werror flag is enabeled.  This
turns all warnings into errors and doens't allow the system to compile
correctly.  Change the system to ignore the 3 warning rules that show up when
compiling.

Upstream-Status: Inappropriate [oe specific - compilation errors due to cross-compiling]

Signed-off-by: Antonios Christidis <a-christidis@ti.com>

---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5cfef6b3..71b6ddb4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -109,7 +109,9 @@ if(CMAKE_COMPILER_IS_GNUCC OR "${CMAKE_CXX_COMPILER_ID}" MATCHES "(Apple)?Clang"
     add_cxx_flag_if_supported(-Wno-error=cpp) # Allow #warning directive
     add_cxx_flag_if_supported(-Wno-unknown-pragmas) # Issue #785
     add_cxx_flag_if_supported(-Wno-error=asm-operand-widths) # Issue #784
-
+    add_cxx_flag_if_supported(-Wno-maybe-uninitialized)
+    add_cxx_flag_if_supported(-Wno-stringop-truncation)
+    add_cxx_flag_if_supported(-Wno-strict-aliasing)
     # -msse -mfpmath=sse to force gcc to use sse for float math,
     # avoiding excess precision problems that cause tests like int2float
     # to falsely fail. -ffloat-store also works, but WG suggested
-- 
2.34.1

